src/main/connector/system/WindowsProcess/WindowsProcess.yaml

```diff
@@ -39,97 +38,28 @@ monitors:
             WriteOperationCount,
-            WriteTransferCount
+            WriteTransferCount,
+            CommandLine
             FROM Win32_Process
-        sqlQuery:
-          type: sql
-          tables:
-          - source: ${source::processQuery}
-            alias: processQuery
-            columns:
-            - name: ProcessId
-              type: INT
-              number: 1
-            - name: CreationDate
-              type: VARCHAR(30)
-              number: 2
-            - name: Name
-              type: VARCHAR(255)
-              number: 3
-            - name: ParentProcessId
-              type: INT
-              number: 4
-            - name: ExecutablePath
-              type: VARCHAR(255)
-              number: 5
-            - name: ThreadCount
-              type: BIGINT
-              number: 6
-            - name: PageFaults
-              type: BIGINT
-              number: 7
-            - name: PageFileUsage
-              type: BIGINT
-              number: 8
-            - name: ReadOperationCount
-              type: BIGINT
-              number: 9
-            - name: ReadTransferCount
-              type: BIGINT
-              number: 10
-            - name: WriteOperationCount
-              type: BIGINT
-              number: 11
-            - name: WriteTransferCount
-              type: BIGINT
-              number: 12
-          query: |
-            SELECT
-              ProcessId, 
-              EXTRACT(EPOCH FROM NOW() AT TIME ZONE 'UTC') - CreationDate AS TimeDifferenceInSeconds,
-              Name,
-              ParentProcessId,
-              ExecutablePath,
-              ThreadCount,
-              PageFaults,
-              PageFileUsage,
-              ReadOperationCount,
-              ReadTransferCount,
-              WriteOperationCount,
-              WriteTransferCount
-            FROM
-              processQuery
-            WHERE
-              Name IN ${var::processNames}
-
-            # The value of the variable 'processNames'
-            # is set in MetricsHub configuration file 'metricshub.yaml'. Here is an example:
-            # resources:
-            #  localhost:
-            #    attributes:
-            #      host.name: localhost:
-            #      host.type: windows
-            #    protocols:
-            #      wmi:
-            #        timeout: 120
-            #    connectors: ["+WindowsProcess"]
-            #    variables:
-            #      WindowsProcess:
-            #        processNames: ('msedge.exe', 'process2.exe')
-
+          computes:
+          - type: keepOnlyMatchingLines
+            column: 2
+            regExp: ${var::nameMatches}
+          - type: keepOnlyMatchingLines
+            column: 12
+            regExp: ${var::commandMatches}   
       mapping:
-        source: ${source::sqlQuery}
+        source: ${source::processQuery}
         attributes:
-          id: $3
-          system.process.path: $5
+          id: $2

```

Comment:
In the attributes, we only want:

* `process.id` (the PID)
* `process.parent.id` (the parent PID)
* `process.match.name: ${var::matchName}`
* `process.match.command: ${var:matchCommand}`

Reply:
Also, `process.path` and `id` should belong to the attributes list. `process.path` is a string and 'id' is a required monitor attribute

Reply:
`process.path` is not something that is usually looked at when listing or monitoring processes. Let's remove it completely. Instead, we will use `process.name`. Thank you!